pgmq-extension/sql/pgmq.sql

```diff
@@ -451,7 +454,9 @@ CREATE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FAL
 RETURNS BOOLEAN AS $$
 DECLARE
     qtable TEXT := pgmq.format_table_name(queue_name, 'q');
+    fq_qtable TEXT := 'pgmq.' || qtable;
     atable TEXT := pgmq.format_table_name(queue_name, 'a');
+    fq_atable TEXT := 'pgmq.' || atable;
```

Comment:
@theory , this is change to `pgmq.drop_queue()`

Reply:
OOOOH, I see, duh.